jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListView.java

```diff
@@ -87,9 +97,11 @@ public String getStyleNames(TaskSummary row,
     @Override
     public void initColumns(ExtendedPagedTable<TaskSummary> extendedPagedTable) {
         initCellPreview(extendedPagedTable);
-
+        
         Column actionsColumn = initActionsColumn();
+        Column<TaskSummary, ?> errorCountColumn = initErrorCountColumn();
```

Comment:
The way is setup in here, the error columns gets visible to be used in the normal _Task List_, that should be avoided so it only appears in the _Task Admin List_